about/MapsontheHill/index.md

```diff
@@ -0,0 +1,23 @@
+# Maps on the Hill
```

Comment:
all new posts need the yaml front matter also. 

Copy and paste it from another one, its the stuff between the 

```yml
---
yada: yada
---
```